mailpoet/tests/unit/EmailEditor/Engine/Renderer/ProcessManagerTest.php

```diff
@@ -39,7 +39,7 @@ public function testItCallsPreprocessorsProperly(): void {
     $secondPreprocessor = $this->createMock(TopLevelPreprocessor::class);
     $secondPreprocessor->expects($this->once())->method('preprocess')->willReturn([]);
 
-    $preprocessManager = new PreprocessManager($cleanup, $topLevel, $blocksWidth, $typography, $spacing);
+    $preprocessManager = new ProcessManager($cleanup, $topLevel, $blocksWidth, $typography, $spacing);
     $preprocessManager->registerPreprocessor($secondPreprocessor);
```

Comment:
The variable name $preprocessManager could be a bit confusing after the renaming

Reply:
This was changed when [I added a post-process class](https://github.com/mailpoet/mailpoet/pull/5428/commits/9d09f72bdf6e2d7817ef5030c2976a0c3c3d2373#diff-f538429edce390b87c33f415a2f4fe8ca5cec7c76bb6ac722a1d446c67422b5eR46-R47).